### PR TITLE
Update to include the same settings as in Eclipse

### PR DESCRIPTION
**Pull request by nateww**
A few more import settings to match what the Eclipse import settings does (primarily related to Play framework imports)

### CODE REVIEW

**Review by AdamJackman — APPROVED** [review]

File: ide-style/intellij/sofi-intellij-code-style.xml [hunk 1]
```diff
@@ -23,6 +23,17 @@
       <emptyLine />
       <package name="com.sofi" withSubpackages="true" static="false" />
       <emptyLine />
+      <package name="configs" withSubpackages="true" static="false" />
```
Comment (BirdHowl) [inline ide-style/intellij/sofi-intellij-code-style.xml:26]: `configs` , `controllers`, `exceptions` and `models` are all grouped together?

Comment (nateww) [reply ide-style/intellij/sofi-intellij-code-style.xml:26]: (shrug) They are usually all related to Play classes, so I left them grouped.

**Review by kspilkington-sofi — APPROVED** [review]

File: ide-style/intellij/sofi-intellij-code-style.xml (hunk 1 above)
Comment (bhowellsofi) [reply ide-style/intellij/sofi-intellij-code-style.xml:26]: But I mean, why are `services` and `views` excluded from that grouping? Don't they also fall into the category of package roots that are usually uniquely provided by play apps for their own logic?

Comment (nateww) [reply ide-style/intellij/sofi-intellij-code-style.xml:26]: I looked some existing codebases, and most of them had services and views broken out with a newline, so this tended to cause the least amount of disturbance in the existing projects.

If you think the addition or removal of emptyLine is of benefit, I have no opinion on adding/removing them.

**Comment by kingpin2k** [comment]
This looks an awful lot like the import PR I tried to push in last November that y'all weren't a fan of ;) https://github.com/SocialFinance/sofi-checks/pull/4

**Comment by bhowellsofi** [comment]
@kingpin2k Sure does.

> that y'all weren't a fan of ;)

I was actually fine with that PR and was planning on merging it, but I think the concerns I stated there are still valid.

----

Reminding me of that... Since we've kind of got a crowd here, maybe we can discuss the merits of our current import style?

In my opinion, our import problems really come from us using a complex and lenient import style that is hard to teach code formatters and newcomers to use. Heck, I still catch people who've been around for a long time making import ordering mistakes. 

Maybe it means the technology is insufficient if it can't format in our import style, while allowing for existing well-formatted code to be untouched and without being tailored to particular situations. 

But maybe it just means that we've made the situation more complicated than it needs to be, since IDEs would very likely have these kinds of capabilities if complex import formatting were a feature that a lot of people think is important?

**Comment by nateww** [comment]
My goal with the above change it have cause less code import/code churn for existing projects.  Othewise, we end up with tons of code churn because folks using Eclipse get different results from folks using IntelliJ, who may/may not be using the standard import order.

At least we're mostly internally consistent with the above.

**Pull request merged**